doc/nrf/introduction.rst

```diff
@@ -3,16 +3,50 @@
 About the |NCS|
 ###############
 
-The |NCS| is hosted, distributed, and supported by Nordic Semiconductor for product development with Nordic SoCs.
-The SDK is distributed in multiple Git repositories with application code, drivers, re-distributable libraries, and forks of open-source code used in the SDK.
+The nRF Connect SDK enables you to develop applications for nRF52, nRF53 and nRF91 Series devices.
+It is a set of open source projects hosted by Nordic Semiconductor, consisting of several components such as:
 
-The main repository is the ``nrf`` repository, which contains application and sample projects as well as the SDK manifest file that enables the repositories to be managed as one code base with the west tool.
+* `sdk-nrf`_ repository.
+* `nrfxlib`_ repository, see the :doc:`nrfxlib documentation <nrfxlib:README>`.
+* `sdk-mcuboot`_ repository, see the :doc:`documentation <mcuboot:index>` in Nordic Semiconductor’s MCUboot fork.
+* `Zephyr`_ project, see the :doc:`documentation <zephyr:index>` in Nordic Semiconductor’s Zephyr fork.
+
+The repositories contain the application code, drivers, re-distributable libraries, and forks of open-source code used in the SDK.
+
+The main repository is ``sdk-nrf``.
+It contains application and sample projects as well as the SDK manifest file that enables you to manage the repositories as one code base with the ``west`` tool.
+
+Toolchain
+*********
+
+The figure below visualizes the toolchain in |NCS|. It is based on the :ref:`Zephyr toolchain <zephyr:getting_started>`.
+It is a set of different tools, each having a role in the creation of an application, from configuring the system to building it.
+
+.. figure:: images/ncs-toolchain.png
+   :alt: NCS toolchain
+
+   NCS toolchain
+
+* :ref:`zephyr:kconfig` generates definitions that configure the whole system.
+* :ref:`Device Tree <zephyr:dt-guide>` describes the hardware.
+* Cmake uses the information from Kconfig and the Device Tree to generate build files, see the `CMake documentation`_.
```

Comment:
```suggestion
* CMake uses the information from Kconfig and the devicetree to generate build files, see the `CMake documentation`_.
```
I think this is misleading. CMake itself has no idea about Kconfig or devicetree, so the CMake documentation will have no information about this.
CMake generates build files based on the provided CMakeList.txt files, which use information from Kconfig and devicetree.